### PR TITLE
Create .NET SDK for AlertsManagement

### PR DESCRIPTION
**Pull request by mahakjain314**

### CODE REVIEW

File: src/SDKs/AlertsManagement/AlertsManagement.Tests/AlertsManagement.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
```
Comment (shahabhijeet) [inline src/SDKs/AlertsManagement/AlertsManagement.Tests/AlertsManagement.Tests.csproj]: nit: format the document

File: src/SDKs/AlertsManagement/AlertsManagement.Tests/AlertsManagement.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+
+  <PropertyGroup>
```
Comment (shahabhijeet) [inline src/SDKs/AlertsManagement/AlertsManagement.Tests/AlertsManagement.Tests.csproj]: @mahakjain314 you do not need this, it's being added implicitly to all test projects.
This way we can control the target Fx version from a central location.

File: src/SDKs/AlertsManagement/Management.AlertsManagement/Microsoft.Azure.AlertsManagement.csproj [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
```
Comment (shahabhijeet) [inline src/SDKs/AlertsManagement/Management.AlertsManagement/Microsoft.Azure.AlertsManagement.csproj]: nit: format the entire xml document

File: src/SDKs/AlertsManagement/Management.AlertsManagement/Microsoft.Azure.AlertsManagement.csproj [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.AlertsManagement</PackageId>
+		<Description>Provides developers with libraries for the Alerts Management</Description>
+		<Version>27.0.0</Version>
```
Comment (shahabhijeet) [inline src/SDKs/AlertsManagement/Management.AlertsManagement/Microsoft.Azure.AlertsManagement.csproj]: your first release is 27.0.0?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please use generate.ps1 to generate your SDK, this will create multiple meta data information that is required for your sdk.

File: src/SDKs/AlertsManagement/AlertsManagement.Tests/AlertsManagement.Tests.csproj (hunk 1 above)
Comment (mahakjain314) [reply src/SDKs/AlertsManagement/AlertsManagement.Tests/AlertsManagement.Tests.csproj]: Fixed

File: src/SDKs/AlertsManagement/AlertsManagement.Tests/AlertsManagement.Tests.csproj (hunk 2 above)
Comment (mahakjain314) [reply src/SDKs/AlertsManagement/AlertsManagement.Tests/AlertsManagement.Tests.csproj]: Makes sense. Removed the same.

File: src/SDKs/AlertsManagement/Management.AlertsManagement/Microsoft.Azure.AlertsManagement.csproj (hunk 3 above)
Comment (mahakjain314) [reply src/SDKs/AlertsManagement/Management.AlertsManagement/Microsoft.Azure.AlertsManagement.csproj]: Fixed

File: src/SDKs/AlertsManagement/Management.AlertsManagement/Microsoft.Azure.AlertsManagement.csproj (hunk 4 above)
Comment (mahakjain314) [reply src/SDKs/AlertsManagement/Management.AlertsManagement/Microsoft.Azure.AlertsManagement.csproj]: My apologies. Changed it to 1.0.0

**Comment by mahakjain314** [comment]
> Please use generate.ps1 to generate your SDK, this will create multiple meta data information that is required for your sdk.

I have pushed the metadata information which was generated. I used this documentation [https://github.com/Azure/azure-rest-api-specs/tree/master/specification/alertsmanagement/resource-manager] as reference to generate the code by making some required changes to spec locally.

**Comment by shahabhijeet** [comment]
@mahakjain314 directory structure has been changed, please open new PR with new directory structure

**Comment by shahabhijeet** [comment]
Please open a new PR against the new directory structure.

**Pull request closed without merging**